src/components/feedback-message/FeedbackMessage.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { WarningIcon, InfoIcon } from '../icon';
+import { Tooltip, TooltipTrigger } from '../tooltip';
+import styles from './FeedbackMessage.css';
+
+const FeedbackMessage = ({ children, type, textColor, iconPosition, variant, tooltipChildren, className }) => {
+    const finalClassNames = classNames(
+        styles.feedbackMessage,
+        styles[type],
+        styles[iconPosition],
+        styles[variant],
+        className
+    );
+
+    const renderTooltip = () => {
+        if (!type) {
+            return;
+        }
+
+        return tooltipChildren ?
+            <TooltipTrigger tooltip={ <Tooltip placement="bottom" style={ { maxWidth: 286 } }>{ tooltipChildren }</Tooltip> }>
```

Comment:
Question: Why is the `maxWidth` defined here?

Reply:
No need of that. I used it for testing and forgot to pass this style as `className`.

Reply:
For no apparent reason. I've moved it to the css file. 👍